request_handlers.cc

```diff
@@ -298,49 +301,49 @@ RequestHandler::Status ReverseProxyRequestHandler::handleRequest(const Request&
     }
     else {
       redirect.AddRequestLine("GET", proxy_path, "HTTP/1.1");
-    } 
+    }
     redirect.AddHeader("Host", redirect_host);
     redirect.AddHeader("Accept", "*/*");
-    redirect.AddHeader("Connection", "close") ; 
+    redirect.AddHeader("Connection", "close") ;
```

Comment:
Haha, it's the little things. Done as part of 0dc29a77f2625be86ef6fa8340b8edef9fe39e74.